source/bluetooth/MicroBitBLEManager.cpp

```diff
@@ -1530,4 +1569,87 @@ static void microbit_dfu_evt_handler(ble_dfu_buttonless_evt_type_t event)
 
 #endif // CONFIG_ENABLED(MICROBIT_BLE_DFU_SERVICE)
 
+
+#define MICROBIT_PANIC_NRF_BASE 70
+#define MICROBIT_PANIC_NRF_FAULT_ID_SD_ASSERT  (MICROBIT_PANIC_NRF_BASE)
+#define MICROBIT_PANIC_NRF_FAULT_ID_APP_MEMACC (MICROBIT_PANIC_NRF_BASE + 1)
+#define MICROBIT_PANIC_NRF_FAULT_ID_SDK_ASSERT (MICROBIT_PANIC_NRF_BASE + 2)
+#define MICROBIT_PANIC_NRF_FAULT_ID_SDK_ERROR  (MICROBIT_PANIC_NRF_BASE + 3)
+#define MICROBIT_PANIC_NRF_FAULT_UNKNOWN       (MICROBIT_PANIC_NRF_BASE + 4)
+
```

Comment:
@finneyj Are these panic number assignments OK?

Reply:
07x is currently free:
https://github.com/lancaster-university/codal-core/blob/master/inc/core/ErrorNo.h#L80

I don't think we can really reserve a code for NRF though - too specific. We could reserve 07x as CPU/SDK specific codeset though. It would make sense to have one of those.

Reply:
So add DEVICE_CPU_SDK = 70 to ErrorNo and use that instead of MICROBIT_PANIC_NRF_BASE?

Reply:
i think keeping NRFxxx is fine here (target repo). But we could reserve 07x in codal-core/inc/errno.h to be CPU specific codes